resources/scripts/create_client.sh

```diff
@@ -0,0 +1,11 @@
+
+if [ $(whoami) == "ubuntu" ] && [ -n "$1" ] ; then
+    mkdir -p ./keys
+	sudo docker-compose run --rm openvpn easyrsa build-client-full $1 nopass
+	sudo docker-compose run --rm openvpn ovpn_getclient $1 > ./keys/$1.ovpn
```

Comment:
Can't we run Docker as non-root?

Reply:
Docker service can run as non-root. This was already tested and works perfectly. Sudo was removed.